### ISSUE TITLE
Firefox extension not available anymore

### ISSUE THREAD

**Issue by jonathan-avecilla** [robindijkhof/overleaf-textarea#83]
The link https://addons.mozilla.org/en-US/firefox/addon/overleaf-textarea/ returns that either the developer or Mozilla removed it. It is still available in Chrome, though. 

It'll be greatly appreciated if you re-upload it to the Firefox add-on page.

**Comment by EmCeBeh** [comment]
Same here, are there plans to resubmit it? Thanks.

**Comment by robindijkhof** [comment]
Not really. I believe Overleaf has a new editor which supports Grammarly. Therefore, I think there is no more need for this plugin. Let me know if I am wrong.
